main.go

```diff
@@ -25,6 +23,18 @@ func main() {
 	app.Version = VERSION
 	app.Usage = "An external resource based scheduler for Rancher."
 	app.Action = run
+	app.Flags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "metadata-address",
+			Usage: "The metadata service address",
+			Value: "rancher-metadata",
+		},
+		cli.StringFlag{
+			Name:  "listen",
+			Usage: "Listen on this port for healthchecks",
+			Value: ":80",
```

Comment:
But the usage doesnt indicate that you can specify a different interface and the log message has 0.0.0.0 in it.
...And from a practical standpoint, I don't think the scheduler is ever going to have more than one interface to listen for healtchecks on.